Invalid interface conversion when running `cscli collections inspect ....`

### What happened?

[root@ip-172-31-13-129 crowdsec]# cscli collections inspect crowdsecurity/base-http-scenarios
INFO[11-07-2022 01:10:23 PM] Patching yaml: '/etc/crowdsec/config.yaml' with '/etc/crowdsec/config.yaml.local' 
type: collections
name: crowdsecurity/base-http-scenarios
filename: base-http-scenarios.yaml
description: 'http common : scanners detection'
author: crowdsecurity
remote_path: collections/crowdsecurity/base-http-scenarios.yaml
version: "0.6"
local_path: /etc/crowdsec/collections/base-http-scenarios.yaml
localversion: "0.6"
localhash: 2d70781df8c630d36e5f4800bde77dd7e130481e9c658aa0b3aae7ae95e15271
installed: true
downloaded: true
uptodate: true
tainted: true
local: false
parsers:
- crowdsecurity/http-logs
scenarios:
- crowdsecurity/http-crawl-non_statics
- crowdsecurity/http-probing
- crowdsecurity/http-bad-user-agent
- crowdsecurity/http-path-traversal-probing
- crowdsecurity/http-sensitive-files
- crowdsecurity/http-sqli-probing
- crowdsecurity/http-xss-probing
- crowdsecurity/http-backdoors-attempts
- ltsich/http-w00tw00t
- crowdsecurity/http-generic-bf
- crowdsecurity/http-open-proxy
collections:
- crowdsecurity/http-cve

Current metrics : 

panic: interface conversion: interface {} is prom2json.Histogram, not prom2json.Metric

goroutine 1 [running]:
main.GetParserMetric({0xc00038dd80, 0x1d}, {0xc000039cf8, 0x17})
	/codebuild/output/src629/src/s3/00/rpmbuild/BUILD/crowdsec-1.4.0/cmd/crowdsec-cli/utils.go:322 +0xc77
main.ShowMetrics(0xc00092b948)
	/codebuild/output/src629/src/s3/00/rpmbuild/BUILD/crowdsec-1.4.0/cmd/crowdsec-cli/utils.go:292 +0x518
main.InspectItem({0x7ffff42ec62d, 0x21}, {0x13d62f3, 0xc00092bc28})
	/codebuild/output/src629/src/s3/00/rpmbuild/BUILD/crowdsec-1.4.0/cmd/crowdsec-cli/utils.go:248 +0x5a6
main.NewCollectionsCmd.func10(0xc000649680, {0xc000113ef0, 0x1, 0x1})
	/codebuild/output/src629/src/s3/00/rpmbuild/BUILD/crowdsec-1.4.0/cmd/crowdsec-cli/collections.go:155 +0x67
github.com/spf13/cobra.(*Command).execute(0xc000649680, {0xc000113ec0, 0x1, 0x1})
	/codebuild/output/src629/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0xc00063a000)
	/codebuild/output/src629/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/codebuild/output/src629/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
main.main()
	/codebuild/output/src629/src/s3/00/rpmbuild/BUILD/crowdsec-1.4.0/cmd/crowdsec-cli/main.go:191 +0xb30


### What did you expect to happen?

No crash

### How can we reproduce it (as minimally and precisely as possible)?

Run `cscli collections inspect ....`

### Anything else we need to know?

_No response_

### Crowdsec version

v1.4.0-rc4

### OS version

Fedora

### Enabled collections and parsers

_No response_

### Acquisition config

_No response_

### Config show

_No response_

### Prometheus metrics

_No response_

### Related custom configs versions (if applicable) : notification plugins, custom scenarios, parsers etc.

_No response_

--- a/cmd/crowdsec-cli/utils.go
+++ b/cmd/crowdsec-cli/utils.go
@@ -319,7 +319,11 @@ func GetParserMetric(url string, itemName string) map[string]map[string]int {
 		}
 		log.Tracef("round %d", idx)
 		for _, m := range fam.Metrics {
-			metric := m.(prom2json.Metric)
+			metric, ok := m.(prom2json.Metric)
+			if !ok {
+				log.Debugf("failed to convert metric to prom2json.Metric")
+				continue
+			}
 			name, ok := metric.Labels["name"]
 			if !ok {
 				log.Debugf("no name in Metric %v", metric.Labels)
@@ -402,7 +406,11 @@ func GetScenarioMetric(url string, itemName string) map[string]int {
 		}
 		log.Tracef("round %d", idx)
 		for _, m := range fam.Metrics {
-			metric := m.(prom2json.Metric)
+			metric, ok := m.(prom2json.Metric)
+			if !ok {
+				log.Debugf("failed to convert metric to prom2json.Metric")
+				continue
+			}
 			name, ok := metric.Labels["name"]
 			if !ok {
 				log.Debugf("no name in Metric %v", metric.Labels)

